[2025-10-20T23:36:47.756+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-20T23:36:47.788+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_Lab1_TaskFlow_FileIO.clean_data_task manual__2025-10-20T23:35:58.452593+00:00 [queued]>
[2025-10-20T23:36:47.797+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_Lab1_TaskFlow_FileIO.clean_data_task manual__2025-10-20T23:35:58.452593+00:00 [queued]>
[2025-10-20T23:36:47.797+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-20T23:36:47.823+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): clean_data_task> on 2025-10-20 23:35:58.452593+00:00
[2025-10-20T23:36:47.837+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=297) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-20T23:36:47.841+0000] {standard_task_runner.py:63} INFO - Started process 330 to run task
[2025-10-20T23:36:47.841+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Airflow_Lab1_TaskFlow_FileIO', 'clean_data_task', 'manual__2025-10-20T23:35:58.452593+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/***.py', '--cfg-path', '/tmp/tmp6cyeh9ay']
[2025-10-20T23:36:47.845+0000] {standard_task_runner.py:91} INFO - Job 13: Subtask clean_data_task
[2025-10-20T23:36:47.984+0000] {task_command.py:426} INFO - Running <TaskInstance: Airflow_Lab1_TaskFlow_FileIO.clean_data_task manual__2025-10-20T23:35:58.452593+00:00 [running]> on host 0074cd054b76
[2025-10-20T23:36:48.523+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='your_name' AIRFLOW_CTX_DAG_ID='Airflow_Lab1_TaskFlow_FileIO' AIRFLOW_CTX_TASK_ID='clean_data_task' AIRFLOW_CTX_EXECUTION_DATE='2025-10-20T23:35:58.452593+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-20T23:35:58.452593+00:00'
[2025-10-20T23:36:48.527+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-20T23:36:48.529+0000] {airflow.py:60} INFO - Cleaning data at /opt/***/dags/data/loaded.pkl
[2025-10-20T23:36:48.645+0000] {python.py:237} INFO - Done. Returned value was: /opt/***/dags/data/cleaned.pkl
[2025-10-20T23:36:48.647+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-20T23:36:48.705+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Airflow_Lab1_TaskFlow_FileIO, task_id=clean_data_task, run_id=manual__2025-10-20T23:35:58.452593+00:00, execution_date=20251020T233558, start_date=20251020T233647, end_date=20251020T233648
[2025-10-20T23:36:48.782+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-20T23:36:48.826+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-20T23:36:48.833+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
